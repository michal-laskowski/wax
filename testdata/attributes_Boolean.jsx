export function attributes_Boolean() {
    return (
        <>
            <input checked={true}></input>
            <input checked={false}></input>
            <input disabled={true}></input>
            <input disabled={false}></input>
            <p hidden translate></p>
            <p hidden={false} translate={false}></p>
            <form novalidate></form>
        </>
    );
}
