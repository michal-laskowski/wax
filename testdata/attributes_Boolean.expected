<input checked>
<input>
<input disabled>
<input>
<p hidden translate></p>
<p></p>
<form novalidate></form>